### PR TITLE
[WFLY-17529] enabled jaxws-retail in parent modules

### PR DESCRIPTION
**Pull request by rsearls**

### CODE REVIEW

File: jaxws-retail/service/pom.xml [hunk 1]
```diff
@@ -39,6 +39,7 @@
     </licenses>
     <properties>
         <checkstyle.skip>true</checkstyle.skip>
+        <version.jbossws-cxf-client>6.1.0.Final</version.jbossws-cxf-client>
```
Comment (emmartins) [inline jaxws-retail/service/pom.xml:42]: Perhaps we should we also add this to the JAXWS Client BOM?

File: jaxws-retail/service/pom.xml [hunk 2]
```diff
@@ -60,6 +61,12 @@
             <scope>provided</scope>
             <type>pom</type>
         </dependency>
+
+        <dependency>
+            <groupId>org.jboss.ws.cxf</groupId>
+            <artifactId>jbossws-cxf-client</artifactId>
+            <version>${version.jbossws-cxf-client}</version>
```
Comment (emmartins) [inline jaxws-retail/service/pom.xml:68]: provided scope or is it supposed for this dep to be packaged in the deployment?

File: jaxws-retail/service/pom.xml [hunk 3]
```diff
@@ -70,6 +77,7 @@
                 <artifactId>maven-war-plugin</artifactId>
                 <configuration>
                     <attachClasses>true</attachClasses>
+                    <packagingExcludes>WEB-INF/lib/cxf-*.jar</packagingExcludes>
```
Comment (emmartins) [inline jaxws-retail/service/pom.xml:80]: I guess this is related to not using the provided scope for the new dependency?

File: jaxws-retail/service/pom.xml (hunk 1 above)
Comment (rsearls) [reply jaxws-retail/service/pom.xml:42]: I have no objection

File: jaxws-retail/service/pom.xml (hunk 2 above)
Comment (rsearls) [reply jaxws-retail/service/pom.xml:68]: Fixed

File: jaxws-retail/service/pom.xml (hunk 3 above)
Comment (rsearls) [reply jaxws-retail/service/pom.xml:80]: Yes

File: jaxws-retail/service/pom.xml (hunk 1 above)
Comment (jbliznak) [reply jaxws-retail/service/pom.xml:42]: it is already there btw https://repo1.maven.org/maven2/org/wildfly/wildfly-jaxws-client-bom/27.0.1.Final/wildfly-jaxws-client-bom-27.0.1.Final.pom

Comment (emmartins) [reply jaxws-retail/service/pom.xml:42]: Indeed, just confirmed such artifact is a top level dependency on the JAXWS-Client BOM (see https://github.com/wildfly/boms/blob/main/client/jaxws-client/pom.xml#L100 ), so no need to manage version here

Comment (rsearls) [reply jaxws-retail/service/pom.xml:42]: org.jboss.ws.plugins:jaxws-tools-maven-plugin does not process wildfly-jaxws-client-bom
as the enclosing pom.xml does, thus org.jboss.ws.cxf:jbossws-cxf-client must be specified
with version in order for wsconsume to function properly.  Declaring the BOM and/or
jbossws-cxf-client in a dependency block within jaxws-tools-maven-plugin does not work
either.  It appears the declaration of jbossws-cxf-client must stay.

**Comment by rsearls** [comment]
@emmartins  There is no conflict in assembly.xml. wfly-17529 and wfly-17528 are separate jiras.  The changes in the assembly.xml is appropriate for each jira.

**Comment by rsearls** [comment]
@emmartins https://github.com/wildfly/boms.git  main branch no longer builds.
I get error
[ERROR] Failed to execute goal org.wildfly.plugins:wildfly-bom-builder-plugin:2.0.5.Final:build-bom (build-bom) on project wildfly-ejb-client-bom-builder: Could not resolve dependencies for project org.wildfly.bom:wildfly-ejb-client-bom-builder:pom:28.0.0.Final-SNAPSHOT: Failed to collect dependencies at io.opentelemetry:opentelemetry-sdk-extension:jar:1.20.0: Failed to read artifact descriptor for io.opentelemetry:opentelemetry-sdk-extension:jar:1.20.0: Could not transfer artifact io.opentelemetry:opentelemetry-sdk-extension:pom:1.20.0 from/to spring-framework-repro (https://repo.spring.io/artifactory/libs-milestone/): Authentication failed for https://repo.spring.io/artifactory/libs-milestone/io/opentelemetry/opentelemetry-sdk-extension/1.20.0/opentelemetry-sdk-extension-1.20.0.pom 401 Unauthorized ->

How to resolve?

**Comment by emmartins** [comment]
Hi @rsearls, I updated the QS repo to use WildFly 28.0.0.Beta1 BOMs and with those I tried this Quickstart without any change, following the README instructions, and build+deploy, wdsl access on browser, and executing the client all worked fine... Can you please confirm that and if all good for you too can you change the PR to just re-enable the QS on build and dist, similar to what was done for jaxws-ejb?

**Comment by rsearls** [comment]
@emmartins 
I am unable to confirm your change for 3 reasons.

  1. I tried a rebase of QS to get your current change.
     There are no changes in https://github.com/jbossws/quickstart.git.
     Git reports upto date.
  2. I tried to pull current changes to https://github.com/wildfly/boms.git
     There were no changes.  And as noted in my comments 3 days ago this
     repo no longer successfully compiles for me.
  3. I find no tagged release 28.0.0.Beta2 in wildfly.

In order to confirm your changes at minimum I should be able to rebase
QS repo and test (1.).   I would prefer to also be able to cleanly build
the wildfly boms repo (2.) as well.

**Comment by emmartins** [comment]
@rsearls all you need is to checkout current main branch of wildfly/quickstart , cd to jaxws-retail and try it with https://github.com/wildfly/wildfly/releases/download/28.0.0.Beta1/wildfly-28.0.0.Beta1.zip (no need to build BOMs or anything else)

**Pull request closed without merging**